schema.ts

```diff
@@ -110,10 +150,33 @@ export interface SliderSchema extends NumberSchema {
   step?: number | string;
 }
 
+export enum IconSet {
```

Comment:
Do we use this icon set for anything other than scales in `coda`? Just wondering if maybe we want to call it ScaleIconSet.

Reply:
It's has other usage in Coda (An example use case is apparently we render the star icon in `paid_feature_banner.tsx`)

Reply:
Hmm ok. Let's call it ScaleIconSet, I'd be surprised if we found another use for this really specific set of icons.

I think in the core product we want to get rid of this tiny subset of icons and use our full icon set, at which point we'd go back to using a string, and probably need to publish our icons8 icon names somewhere.

Reply:
yeah sounds good. It did seem a little weird for the non scale usages in coda repo.